doc/sphinx/practical-tools/utilities.rst

```diff
@@ -563,9 +595,45 @@ To build, say, two targets foo.vo and bar.vo in parallel one can use
   utility, in particular it may build a shared prerequisite twice.
 
 
+The grammar of _CoqProject
+++++++++++++++++++++++++++
+A _CoqProject file is implicitly assumed to be encoded in UTF-8. The parser of
+``coq_makefile`` uses the following grammar:
```

Comment:
```suggestion
``coq_makefile`` uses the following grammar:
```

No need to mention UTF-8, either.

Reply:
The only difference in semantics between linebreaks and other blanks is, that linebreaks terminate comments. And this behaviour is, I think, already explained.